misc/change_leasetime

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+#
+# Dynamically increase the DHCP lease time, requires killing and restarting
+# dnsmasq as per documentation (SIGHUP does not reload configuration file).
+lease=$1
+if [ -z $lease ]; then
+    lease="500s"
```

Comment:
I would make this error out -- no reason to have a default here and it can only cause confusion.